pages/utilities/css-units-converter.tsx

```diff
@@ -0,0 +1,207 @@
+    const containerWidthValue = parseFloat(widthInput);
+    const containerHeightValue = parseFloat(heightInput);
+
+    if (isNaN(value) || (needsContainerInput && isNaN(containerWidthValue))) {
+      setOutput("Invalid values.");
+      return;
+    }
+
+    const conversionKey = `${fromUnit}-${toUnit}` as ConversionKey;
+    const conversionFunction = conversionFunctionMapper[conversionKey];
+
+    if (!(conversionKey in conversionFunctionMapper)) {
+      setOutput("Invalid conversion");
+      return;
+    }
+
+    let convertedValue: string | number = "Invalid conversion";
+
+    if (conversionFunction) {
+      if (["px-vw", "vw-px"].includes(conversionKey)) {
+        convertedValue = conversionFunction(value, containerWidthValue);
+      } else if (["px-vh", "vh-px"].includes(conversionKey)) {
+        convertedValue = conversionFunction(value, containerHeightValue);
+      } else if (["px-vmin", "vmin-px"].includes(conversionKey)) {
+        convertedValue = conversionFunction(
+          value,
+          containerWidthValue,
+          containerHeightValue
+        );
+      } else if (["px-vmax", "vmax-px"].includes(conversionKey)) {
+        convertedValue = conversionFunction(
+          value,
+          containerWidthValue,
+          containerHeightValue
+        );
+      } else {
+        convertedValue = conversionFunction(value);
+      }
+
+      setOutput(convertedValue.toString());
+    }
+  }, [
+    fromUnit,
+    heightInput,
+    htmlInput,
+    needsContainerInput,
+    toUnit,
+    widthInput,
+  ]);
+
+  return (
+    <main>
+      <Meta
+        title="CSS units converter by Jam.dev | Free, Open Source & Ad-free"
+        description="Convert your css units values to other options"
+      />
+      <Header />
+      <CMDK />
+      <section className="container max-w-2xl mb-12">
+        <PageHeader
+          title="CSS Units Converter"
+          description="Convert your css units values to other options"
+        />
+      </section>
+      <section className="container max-w-2xl mb-6">
+        <Card className="flex flex-col p-6 hover:shadow-none shadow-none rounded-xl">
+          <div>
+            <Label>Input Value</Label>
+            <Textarea
+              className="mb-6"
+              rows={3}
+              value={htmlInput}
+              onChange={(e) => setHtmlInput(e.target.value)}
+            />
+            <div className="mb-6 flex w-full gap-4">
+              <div className="flex flex-1 flex-col">
+                <Label>From Unit</Label>
+                <Combobox
+                  data={unitOptions}
+                  onSelect={(value) => setFromUnit(value)}
+                />
+              </div>
+              <div className="flex flex-1 flex-col">
+                <Label>To Unit</Label>
+                <Combobox
+                  data={unitOptions}
+                  onSelect={(value) => setToUnit(value)}
+                />
+              </div>
```

Comment:
Let's have `defaultValue` for Combobox. I believe pixel to rem is most popular option.
I would suggest using it as default value.

Like this:

<img width="647" alt="image" src="https://github.com/user-attachments/assets/c8d08344-4394-4643-8b9e-285021564040">